src/backend/TrafficCourts/Citizen.Service/Controllers/DisputesController.cs

```diff
@@ -178,17 +178,20 @@ public async Task<IActionResult> VerifyEmailAsync([Required] string token, Cance
             var response = await _bus.Request<SearchDisputeRequest, SearchDisputeResponse>(message, cancellationToken);
             IActionResult result;
 
-            if (!string.IsNullOrEmpty(response.Message.DisputeId))
+            if (!string.IsNullOrEmpty(response.Message.NoticeOfDisputeGuid))
             {
                 _logger.LogDebug("Dispute found");
-                var disputeId = _hashids.EncodeHex(response.Message.DisputeId.PadLeft(48, '0'));
+                var identifier = _hashids.EncodeHex(response.Message.NoticeOfDisputeGuid);
                 _ = Enum.TryParse(response.Message.DisputeStatus, out DisputeStatus disputeStatus);
                 _ = Enum.TryParse(response.Message.JJDisputeStatus, out JJDisputeStatus jjDisputeStatus);
+                _ = Enum.TryParse(response.Message.HearingType, out JJDisputeHearingType hearingType);
+
                 SearchDisputeResult searchResult = new()
                 {
-                    DisputeId = disputeId,
+                    Identifier = identifier,
                     DisputeStatus = disputeStatus,
-                    JJDisputeStatus = jjDisputeStatus
+                    JJDisputeStatus = jjDisputeStatus,
+                    HearingType = hearingType

```

Comment:
Is this a new requirement? HearingType is not in the TCVP-1833 spec so this data will not get returned currently.

Reply:
Yes, that was not in the TCVP-1833 spec, we found we need to add that in the last sprint demo. I think we will have a new ticket for this endpoint, just getting a temporary field for frontend now